README.md

```diff
@@ -47,17 +47,23 @@ echo "XYZ [${XYZ}]"
 DATE=$(git show -s --format=%ci upstream/master | cut -d" " -f1)
 echo "DATE [${DATE}]"
 
-SHA=$(git show -s --format=%h upstream/master)
-echo "SHA [${SHA}]"
+vtkSHA=$(git show -s --format=%h upstream/master)
+echo "vtkSHA: [${SHA}]"
 
-BRANCH_NAME=slicer-v${XYZ}-${DATE}-${SHA}
+BRANCH_NAME=slicer-v${XYZ}-${DATE}-${vtkSHA}
 echo "BRANCH_NAME [${BRANCH_NAME}]"
 
-git checkout -b ${BRANCH_NAME} ${SHA}
+git checkout -b ${BRANCH_NAME} ${vtkSHA}
```

Comment:
For sake of consistency, it would also be nice to update the other fork we managed similarly:

https://github.com/Slicer/ITK
https://github.com/Slicer/SimpleITK


There are few other one that are now tracked using the GitHub topic [3d-slicer-fork](https://github.com/topics/3d-slicer-fork)

Idea: Instead of duplicating instruction on each fork README, may we could have a gist (or project named `Slicer-fork-guideline`?)

Reply:
> Instead of duplicating instruction on each fork README, may we could have a gist (or project named Slicer-fork-guideline?)

Sounds good. I don't think there is a way to allow shared editing of gist, so either project or maybe wiki/docs page (in the new doc system).